### PR TITLE
Branding Page

### PR DESCRIPTION
**Pull request by codiejay**
- [x] Create static page for branding 
- [x] Import brand images 
- [x] Implement download as .png and .svg 

[Page link](https://iq-wiki-git-831-branding-page-for-iqwiki-prediqt.vercel.app/static/branding)

closes https://github.com/EveripediaNetwork/issues/issues/831

### CODE REVIEW

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
one of the cards on the third row seems to be broken
image

there should be options to download as png/svg, both means transparency. and the background shouldn't be downloaded together. as I earlier mentioned, they are separate entities.

There is an issue on top of the page too
image

looking good so far

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
There is no way to go back to the log after clicking on download. I designed with an intention of the the download options to be an overlay. 
what was implemented is a good idea, but it would be nice if logos can go back to original position on click
image

I don't know why this part is missing
image

on tablet and mobile, the spacing at the very top is too small.
image
great job so far

**Review by Simplypheyie96 — APPROVED** [review]
Awesome work!!

**Comment by Emmanueldmlr** [comment]
@codiejay, why don't you want to use SVGs instead of .png?

File: src/components/Branding/BrandingAssets.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  useEffect(() => {
+    if (currentlyViewed === bg.bg) {
+      setShowDownloadOptions(true)
+    } else {
+      setShowDownloadOptions(false)
+    }
+  }, [currentlyViewed, bg.bg])
+  return (
+    <Flex
+      w={{ lg: '30%', base: '45%', md: '30%' }}
```
Comment (Emmanueldmlr) [inline src/components/Branding/BrandingAssets.tsx:25]: Do this instead
```suggestion
      w={{ base: '45%', md: '30%' }}
```

File: src/pages/static/branding.tsx [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+      download: '/branding/downloadassets/logobrainwhitenegative',
+    },
+    {
+      bg: '/branding/braindaoBlackNegative.png',
+      download: '/branding/downloadassets/logobrainblacknegative',
+    },
+  ]
+  return (
+    <Box bg="#E2E8F0" _dark={{ backgroundColor: '#1A202C' }} pb={20}>
+      <Box maxW={{ base: '90%', xl: '90%', '2xl': '1280px' }} mx="auto">
```
Comment (Emmanueldmlr) [inline src/pages/static/branding.tsx:97]: do this instead
```suggestion
      <Box maxW={{ base: '90%', '2xl': '1280px' }} mx="auto">
```

**Review by Emmanueldmlr — CHANGES_REQUESTED** [review]

**Comment by Emmanueldmlr** [comment]
@codiejay, kindly add the link to the branding page to the PR so I can check the page out

File: src/pages/static/branding.tsx [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+      bg: '/branding/braindaowhiteNegative.png',
+      download: '/branding/downloadassets/logobrainwhitenegative',
+    },
+    {
+      bg: '/branding/braindaoBlackNegative.png',
+      download: '/branding/downloadassets/logobrainblacknegative',
+    },
+  ]
+  return (
+    <Box bg="#E2E8F0" _dark={{ backgroundColor: 'tetiaryDark' }} pb={20}>
```
Comment (Emmanueldmlr) [inline src/pages/static/branding.tsx:96]: You missed out the color code here.

**Review by Emmanueldmlr — APPROVED** [review]

**Comment by kesar** [comment]
this is not linked from any menu? I guess it should be in footer and perhaps navbar under learn section

**Pull request merged**